fiftyone/plugins/skills.py

```
@@ -0,0 +1,232 @@
+                shutil.rmtree(link_path)
+
+            rel_target = os.path.relpath(skill_central, agent_skills_dir)
+            os.symlink(rel_target, link_path)
+            logger.info(f"Created symlink '{link_path}' -> '{rel_target}'")
+
+        installed.append(skill.name)
+
+    return installed
+
+
+def _iter_plugin_skills(plugin_definition):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_iter_plugin_skills`

(ANN202)